alignak/objects/item.py

```diff
@@ -155,6 +155,7 @@ def __init__(self, params=None, parsing=True):
                               "it is not in %s object properties" % \
                               (key, cls.__name__)
                     self.configuration_warnings.append(warning)
+                    self.properties[key] = ToGuessProp(default='')
```

Comment:
We need to add it to propertie dict in order to get this property serialized later on